apps/api/v2/src/modules/routing-forms-responses/services/routing-forms-responses.service.ts

```diff
@@ -0,0 +1,17 @@
+import { RoutingFormsResponsesOutputService } from "@/modules/routing-forms-responses/services/routing-forms-responses-output.service";
+import { Injectable } from "@nestjs/common";
+
+import { RoutingFormsResponsesRepository } from "../routing-forms-responses.repository";
+
+@Injectable()
+export class RoutingFormsResponsesService {
+  constructor(
+    private readonly routingFormsRepository: RoutingFormsResponsesRepository,
+    private readonly routingFormsResponsesOutputService: RoutingFormsResponsesOutputService
+  ) {}
+
+  async getRoutingFormResponses(routingFormId: string) {
+    const responses = await this.routingFormsRepository.getRoutingFormResponses(routingFormId);
+    return this.routingFormsResponsesOutputService.getRoutingFormResponses(responses);
```

Comment:
nit: They both have same name and very different purpose.
One fetches from DB and later parses it. Maybe we can call the later one, `parseRoutingFormResponses` or `buildRoutingFormResponses`